test/Libraries/RevitTestServices/RevitNodeTestBase.cs

```diff
@@ -43,8 +43,7 @@ public override void Setup()
 
         protected override TestSessionConfiguration GetTestSessionConfiguration()
         {
-            var asmDir = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
-            return new TestSessionConfiguration(Path.GetFullPath(asmDir + @"\..\"), asmDir);
+            return new TestSessionConfiguration(Dynamo.Applications.DynamoRevitApp.DynamoCorePath);
```

Comment:
I am curious on this as well because I have not been able to ask RTF to launch Revit and Dynamo regardless of this change. Any idea here @sharadkjaiswal